web/src/pages/index.md

```diff
@@ -7,35 +7,55 @@ title: ECE 477 Team 5 Homepage
 
 ## Project Description
 
-The project at hand involves the development of a smart air hockey table that
-replicates the functionalities of a traditional arcade air hockey setup with a
-twist. The notable innovation is the incorporation of a responsive surface
-beneath a translucent playfield. This reactive surface, comprising strategically
-positioned lights, sensors, and computational components, reacts in real-time to
-the movements of the puck, generating visual displays that enhance the gameplay
-experience.
-
-The dynamic nature of the reactive surface introduces novel game modes that
-transcend the boundaries of conventional air hockey. Despite its technologically
-advanced features, the smart air hockey table maintains a compact form factor,
-making it adaptable to a variety of flat surfaces, from regular tables to other
-suitable platforms. This convergence of timeless gameplay with modern
-technological advancements is poised to redefine the enjoyment of air hockey,
-offering enthusiasts an immersive and unparalleled gaming encounter.
+The Smart Air Hockey Table (SAHT) redefines the classic arcade game, offering
+an immersive experience like no other. Central to its advanced gameplay is a
+vibrant grid of RGB LEDs, dynamically illustrating the puck's position and game
+events. This real-time interactivity is made possible by a carefully arrayed set
+of hall effect sensors, which pinpoint the puck's every move. Additionally,
+photoresistors are integrated for impeccable goal detection, ensuring each score
+is accurately registered.
+
+The hall effect sensors are strategically embedded beneath the translucent
+playing field in a grid pattern. These sensors detect changes in magnetic
+fields. As the puck, equipped with a small magnet, glides across the surface,
+the sensors underneath identify its precise location by sensing the magnetic
+field changes caused by the puck's movement. The density and layout of this grid
+ensure the puck's position is determined with high accuracy, allowing the system
+to provide real-time feedback. For instance, as the puck moves, the individually
+controlled LEDs beneath the surface illuminate its path, creating a mesmerizing
+trail effect that enhances the overall gaming experience.
+
+This grid-based hall effect sensor system not only offers unparalleled precision
+in puck tracking but also enables the microcontroller to interpret rapid changes
+in direction and speed. Such granular detail ensures that dynamic visuals
+generated by the LEDs are always synchronized with the puck's position and
+movement. Players, therefore, experience a highly responsive and visually rich
+gameplay, where the table reacts instantly to every twist and turn of the puck.
+
+Combined with other features like the OLED score display and photoresistor-based
```

Comment:
"other features like the OLED score display and photoresistor-based goal detection," but the OLED display is never mentioned anywhere else?

Reply:
photoresistors are mentioned, first paragraph. forgot to re-add the OLED to first paragraph lol.